Jenkinsfile

```diff
@@ -522,15 +511,23 @@ EOL
                         }
                     }
                 }
-                stage('Upload Bare models') {
+                stage("Build and upload test models") {
```

Comment:
worth putting this step in parallel with other build stages, because push stages in this section take just a few seconds and this build will add extra time in result.